src/components/CanvasSaveButton/CanvasSaveButton.js

```diff
@@ -0,0 +1,150 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import OLMap from 'ol/Map';
+import Button from '../Button';
+
+const propTypes = {
+  /**
+   * Title of the button.
+   */
+  title: PropTypes.string.isRequired,
+
+  /**
+   * Children content of the button.
+   */
+  children: PropTypes.element.isRequired,
+
+  /**
+   * CSS class of the button.
+   */
+  className: PropTypes.string,
+
+  /**
+   * Translation function.
+   */
+  t: PropTypes.func,
+
+  /**
+   * HTML tabIndex attribute
+   */
+  tabIndex: PropTypes.number,
+
+  /**
+   * Format to save the image in ('image/jpeg' or 'image/png')
+   */
+  saveFormat: PropTypes.string,
+
+  /** An existing [ol/Map](https://openlayers.org/en/latest/apidoc/module-ol_Map-Map.html). */
+  map: PropTypes.instanceOf(OLMap),
+};
+
+const defaultProps = {
+  map: null,
+  tabIndex: 0,
+  className: 'tm-share-button',
```

Comment:
We are not in Share component anymore, so use the class `tm-canvas-save-button`.